python/gtsam/tests/test_GraphvizFormatting.py

```diff
@@ -0,0 +1,139 @@
+"""
+GTSAM Copyright 2010-2021, Georgia Tech Research Corporation,
+Atlanta, Georgia 30332-0415
+All Rights Reserved
+
+See LICENSE for the license information
+
+Unit tests for Graphviz formatting of NonlinearFactorGraph.
+Author: Frank Dellaert & senselessDev
```

Comment:
I'm not an author :-)

Reply:
Well, I did not write the copyright section ;)

Reply:
You don't need that copyright here, unless you're affiliated with georgia tech. Feel free to remove or replace.

Reply:
No, I'm not affiliated with GT. That will trigger the CI pipeline once again, right? Is that worth it?

Reply:
It is and I recommend adding your full name and email so you can be contacted in the future if need be.

Reply:
Won't do, sorry. But changed it nevertheless.